sdk/storage/Azure.Storage.Common/src/Constants.cs

```diff
@@ -33,6 +34,283 @@ internal static class Constants
         /// </summary>
         public const string BlockNameFormat = "Block_{0:D5}";
 
+        // SASTimeFormat represents the format of a SAS start or expiry time. Use it when formatting/parsing a time.Time.
+        // ISO 8601 uses "yyyy'-'MM'-'dd'T'HH':'mm':'ss"
+        public const string TimeFormat = "yyyy-MM-ddTHH:mm:ssZ";
+
+        public const string SnapshotParameterName = "snapshot";
+
+        /// <summary>
+        /// The default port numbers for development storage credentials
+        /// </summary>
+        public const int blobEndpointPortNumber = 10000;
+        public const int queueEndpointPortNumber = 10001;
+        public const int tableEndpointPortNumber = 10002;
+
+        /// <summary>
+        /// Storage Connection Strings 
+        /// </summary>
+        internal static class ConnectionStrings
+        {
+            internal const string UseDevelopmentSetting = "UseDevelopmentStorage";
+            internal const string DevelopmentProxyUriSetting = "DevelopmentStorageProxyUri";
+            internal const string DefaultEndpointsProtocolSetting = "DefaultEndpointsProtocol";
+            internal const string AccountNameSetting = "AccountName";
+            internal const string AccountKeyNameSetting = "AccountKeyName";
+            internal const string AccountKeySetting = "AccountKey";
+            internal const string BlobEndpointSetting = "BlobEndpoint";
+            internal const string QueueEndpointSetting = "QueueEndpoint";
+            internal const string TableEndpointSetting = "TableEndpoint";
+            internal const string FileEndpointSetting = "FileEndpoint";
+            internal const string BlobSecondaryEndpointSetting = "BlobSecondaryEndpoint";
+            internal const string QueueSecondaryEndpointSetting = "QueueSecondaryEndpoint";
+            internal const string TableSecondaryEndpointSetting = "TableSecondaryEndpoint";
+            internal const string FileSecondaryEndpointSetting = "FileSecondaryEndpoint";
+            internal const string EndpointSuffixSetting = "EndpointSuffix";
+            internal const string SharedAccessSignatureSetting = "SharedAccessSignature";
+            internal const string DevStoreAccountName = "devstoreaccount1";
+            internal const string DevStoreAccountKey = "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==";
```

Comment:
What is this string used for?

Reply:
There's a well known [account name/key for the emulator.](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator#authorize-with-shared-key-credentials)